src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -607,7 +610,7 @@ final case class PurposeApiServiceImpl(
           eserviceId = eserviceId,
           consumerId = purpose.consumerId,
           riskAnalysisForm = purpose.riskAnalysisForm.map(riskAnalysisToSeed),
-          title = s"${purpose.title} - clone",
+          title = s"${purpose.title} - clone - ${dateTimeSupplier.get().format(dtf)}",
```

Comment:
The new method to define clone title is described [here](https://github.com/pagopa/interop-be-monorepo/blob/374aad1b15d4df3841b67bc292361d40175a13d2/packages/purpose-process/src/services/purposeService.ts#L804)